core/src/main/scala/ai/lum/odinson/lucene/index/IncrementalOdinsonIndex.scala

```diff
@@ -20,12 +20,15 @@ import org.apache.lucene.index.{
 }
 import org.apache.lucene.search.highlight.TokenSources
 import org.apache.lucene.search.{
+  BooleanClause,
+  BooleanQuery,
   Collector,
   CollectorManager,
   IndexSearcher,
-  Query,
   SearcherManager,
-  TopDocs
+  TopDocs,
+  TermQuery,
+  Query
```

Comment:
Isn't the formatter supposed to alphabetize these?

Reply:
It's capable of doing that via [`rewrite.imports.sort = scalastyle`](https://scalameta.org/scalafmt/docs/configuration.html#imports-sort--scalastyle), but apparently we haven't included that in the current configuration.  Maybe we should, but that's probably better left for a separate PR.